src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -317,297 +331,303 @@ describe('Disables proctoring provider options', () => {
-    expect(spinner.textContent).toEqual('Loading...');
-  });
+      axiosMock = new MockAdapter(getAuthenticatedHttpClient());
+      axiosMock.onGet(StudioApiService.getProctoredExamSettingsUrl(defaultProps.courseId)).reply(200, {
+        proctored_exam_settings: {
+          enable_proctored_exams: true,
+          allow_proctoring_opt_out: false,
+          proctoring_provider: 'mockproc',
+          proctoring_escalation_email: 'test@example.com',
+          create_zendesk_tickets: true,
+        },
+        available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
+        course_start_date: '2070-01-01T00:00:00Z',
+      });
+    }
+
+    it('Hides opting out and zendesk tickets for non edX staff', async () => {
+      setup(false);
+      await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
+      expect(screen.queryByTestId('allowOptingOutYes')).toBeNull();
+      expect(screen.queryByTestId('createZendeskTicketsYes')).toBeNull();
+    });
+
+    it('Shows opting out and zendesk tickets for edX staff', async () => {
+      setup(true);
+      await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
+      expect(screen.queryByTestId('allowOptingOutYes')).not.toBeNull();
+      expect(screen.queryByTestId('createZendeskTicketsYes')).not.toBeNull();
+    });
+  });
+
+  describe('Connection states', () => {
+    beforeEach(() => {
+      initializeMockApp({
+        authenticatedUser: {
+          userId: 3,
+          username: 'abc123',
+          administrator: true,
+          roles: [],
+        },
+      });
+      axiosMock = new MockAdapter(getAuthenticatedHttpClient());
+    });
 
-  it('show connection error message when we suffer server side error', async () => {
-    const errorObject = {
-      customAttributes: {
-        httpErrorStatus: 500,
-      },
-    };
-    auth.getAuthenticatedHttpClient = jest.fn(() => ({
-      get: async () => {
-        throw errorObject;
-      },
-    }));
+    it('Shows the spinner before the connection is complete', async () => {
+      await act(async () => {
+        render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />));
+        // This expectation is _inside_ the `act` intentionally, so that it executes immediately.
+        const spinner = screen.getByTestId('spinnerContainer');
+        expect(spinner.textContent).toEqual('Loading...');
+      });
+    });
 
-    await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
-    const connectionError = screen.getByTestId('connectionError');
-    expect(connectionError.textContent).toEqual(
-      expect.stringContaining('We encountered a technical error'),
-    );
-  });
+    it('Show connection error message when we suffer server side error', async () => {
+      axiosMock.onGet(
+        StudioApiService.getProctoredExamSettingsUrl(defaultProps.courseId),
+      ).reply(500);
```

Comment:
So check this out!  We just reply with a 500 error and no longer have to set up an errorObject or manually throw an error because of the change over in ProctoredExamSettings.